crates/autopilot/src/infra/persistence/auction/mod.rs

```diff
@@ -0,0 +1,46 @@
+use {crate::domain, tracing::Instrument};
+
+pub mod dto;
+
+impl super::Persistence {
+    /// Saves the given auction to storage for debugging purposes.
+    ///
+    /// There is no intention to retrieve this data programmatically.
+    pub fn archive_auction(&self, id: domain::AuctionId, instance: &domain::Auction) {
```

Comment:
Why do we need to differentiate between replace and archive current auction? Can't we just write it to the DB and store it on S3 in the same call? From the domain's perspective, there is no real differentiation between the two is there?

Reply:
done